src/query/service/src/global_services.rs

```diff
@@ -105,7 +108,8 @@ impl GlobalServices {
             global_services.clone(),
         )
         .await?;
-        RoleCacheManager::init(global_services.clone())
+        RoleCacheManager::init(global_services.clone())?;
+        GlobalConfig::init(global_services)
```

Comment:
Should `GlobalConfig` needs to be init as the first entry of global services?

Reply:
I think this doesn't matter?